frontend/src/metabase/browse/components/BrowseDatabases.tsx

```diff
@@ -30,34 +36,76 @@ export const BrowseDatabases = ({
     return <LoadingAndErrorWrapper loading />;
   }
 
-  return databases?.length ? (
-    <DatabaseGrid data-testid="database-browser">
-      {databases.map(database => (
-        <div key={database.id}>
-          <DatabaseCardLink to={Urls.browseDatabase(database)}>
-            <DatabaseCard>
-              <Icon
-                name="database"
-                color={color("accent2")}
-                className={CS.mb3}
-                size={32}
-              />
-              <Title order={2} size="1rem" lh="1rem" color="inherit">
-                {database.name}
-              </Title>
-            </DatabaseCard>
-          </DatabaseCardLink>
-        </div>
-      ))}
-    </DatabaseGrid>
-  ) : (
-    <CenteredEmptyState
-      title={<Box mb=".5rem">{t`No databases here yet`}</Box>}
-      illustrationElement={
-        <Box mb=".5rem">
-          <img src={NoResults} />
-        </Box>
-      }
-    />
+  if (!databases?.length) {
+    return (
+      <CenteredEmptyState
+        title={<Box mb=".5rem">{t`No databases here yet`}</Box>}
+        illustrationElement={
+          <Box mb=".5rem">
+            <img src={NoResults} />
+          </Box>
+        }
+      />
+    );
+  }
+
+  return (
+    <BrowseContainer>
+      <BrowseHeader>
+        <BrowseSection>
+          <Flex w="100%" direction="row" justify="space-between" align="center">
+            <Title order={1} color="text-dark">
+              <Group spacing="sm">
+                <Icon color={color("brand")} name="database" />
+                {t`Databases`}
+              </Group>
+            </Title>
+            <LearnAboutDataLink />
+          </Flex>
+        </BrowseSection>
+      </BrowseHeader>
+      <BrowseMain>
+        <BrowseSection>
+          <DatabaseGrid data-testid="database-browser">
+            {databases.map(database => (
+              <div key={database.id}>
+                <DatabaseCardLink to={Urls.browseDatabase(database)}>
+                  <DatabaseCard>
+                    <Icon
+                      name="database"
+                      color={color("accent2")}
+                      className={CS.mb3}
+                      size={32}
+                    />
+                    <Title order={2} size="1rem" lh="1rem" color="inherit">
+                      {database.name}
+                    </Title>
+                  </DatabaseCard>
+                </DatabaseCardLink>
+              </div>
+            ))}
+          </DatabaseGrid>
+        </BrowseSection>
+      </BrowseMain>
+    </BrowseContainer>
   );
 };
+
+const LearnAboutDataLink = () => (
+  <Flex
+    ml="auto"
+    p=".75rem"
+    justify="right"
+    align="center"
+    style={{ flexBasis: "40.0%" }}
+  >
```

Comment:
Is this flex actually required? I think the parent component specifies `space-between`, and this element only has 1 child

Reply:
This is code that I moved over from `frontend/src/metabase/browse/components/BrowseApp.tsx` (deleted in this branch), so I figure I may as well leave it.